scripts/qa_crowbarsetup.sh

```diff
@@ -479,26 +481,37 @@ function onadmin_prepare_sles_repos()
     local targetdir_install="$tftpboot_suse_dir/install"
 
     if [ -n "${localreposdir_target}" ]; then
-        add_mount "SUSE-Cloud-SLE-11-SP3-deps/sle-11-x86_64/" "" "${targetdir_install}" "Cloud-Deps"
+        add_mount "SUSE-Cloud-SLE-11-SP3-deps/sle-11-x86_64/" "" \
+            "${targetdir_install}" "Cloud-Deps"
         zypper_refresh
     else
-        zypper se -s sles-release|grep -v -e "sp.up\s*$" -e "(System Packages)" |grep -q x86_64 || zypper ar http://$susedownload/install/SLP/SLES-${slesversion}-LATEST/x86_64/DVD1/ sles
+        zypper se -s sles-release | \
+            grep -v -e "sp.up\s*$" -e "(System Packages)" | \
+            grep -q x86_64 \
+        || zypper ar \
+            http://$susedownload/install/SLP/SLES-${slesversion}-LATEST/x86_64/DVD1/ \
+            sles
 
         if ! $longdistance ; then
-            add_mount "" "clouddata.cloud.suse.de:/srv/nfs/suse-$suseversion/install" "${targetdir_install}"
+            add_mount "" \
+                "clouddata.cloud.suse.de:/srv/nfs/suse-$suseversion/install" \
+                "$targetdir_install"
         fi
 
         local repo
         for repo in $slesrepolist ; do
             local zypprepo=""
             [ "$WITHSLEUPDATES" != "" ] && zypprepo="$repo"
-            add_mount "$zypprepo" "clouddata.cloud.suse.de:/srv/nfs/repos/$repo" \
+            add_mount "$zypprepo" \
+                "clouddata.cloud.suse.de:/srv/nfs/repos/$repo" \
                 "$tftpboot_repos_dir/$repo"
         done
 
         # just as a fallback if nfs did not work
         if [ ! -e "$targetdir_install/media.1/" ]; then
             download_and_mount_sles "$tftpboot_suse_dir" "$targetdir_install"
+                http://$susedownload/install/SLES-$slesversion-$slesmilestone/$f \
+            || complain 70 "iso not found"
```

Comment:
Sorry :-(  This kind of thing can happen when PRs are open for a long time.  Fixed now.